test/DocumentFormat.OpenXml.Tests/ofapiTest/OpenXmlReaderTest.cs

```diff
@@ -538,5 +539,66 @@ private void TestReaderOnParagraph(OpenXmlReader targetReader)
             Assert.Equal(typeof(Paragraph), targetReader.ElementType);
             Assert.True(string.IsNullOrEmpty(targetReader.GetText()));
         }
+
+        ///<summary>
+        ///  A test for the LoadElement generic method.
+        ///</summary>
+        [Fact]
+        public void SuccessfulLoadElementGeneric()
```

Comment:
Can you add some negative tests? Tests to verify that exceptions are thrown when expected?